lib/line/bot/client.rb

```diff
@@ -145,6 +145,25 @@ def revoke_channel_access_token_jwt(access_token)
         post(oauth_endpoint, endpoint_path, payload, headers)
       end
 
+      # Verify ID token
+      #
+      # @param id_token [String] ID token
+      # @param options [Hash] Optional request body
+      #
+      # @return [Net::HTTPResponse]
+      def verify_id_token(id_token, options = {})
```

Comment:
Passing options as a hash may result in unintended parameters being passed to the endpoint. It may be too much to worry about, but it is better to be safe than sorry.

I think it would be better to pass only concrete parameters, i.e. `nonce` and `user_id`, like other methods such as narrowcast.
https://github.com/line/line-bot-sdk-ruby/blob/8760addfd657301e1b53aa7421e6d0d6595cf908/lib/line/bot/client.rb#L257

```suggestion
      def verify_id_token(id_token, nonce: nil, user_id: nil)
```